Sources/IBLinterKit/Rules/HidesBottomBarRule.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  HidesBottomBarRule.swift
+//  IBLinterKit
+//
+//  Created by ykkd on 2022/02/06.
+//
+
+import IBDecodable
+
+extension Rules {
+
+    struct HidesBottomBarRule: Rule {
+
+        static var identifier = "hides_bottom_bar"
+        static var description = "Check if hidesBottomBarWhenPushed is enabled"
+
+        private let excluded: [String]
+
+        init(context: Context) {
+            excluded = context.config.hidesBottomBarRule?.excludedViewControllers ?? []
+        }
+
+        func validate(xib: XibFile) -> [Violation] { return [] }
+
+        func validate(storyboard: StoryboardFile) -> [Violation] {
+            guard let scenes = storyboard.document.scenes else { return [] }
+            let viewControllers = scenes.compactMap { $0.viewController?.viewController }
+            return viewControllers.compactMap { validate(for: $0, file: storyboard) }
+        }
+
+        private func validate<T: InterfaceBuilderFile>(for viewController: ViewControllerProtocol, file: T) -> Violation? {
+            guard let customClass = viewController.customClass,
```

Comment:
It seems this rule is only applied for view controllers with custom classes, is this intentional? Should we warn a VC, which doesn't specify hidesBottomBarWhenPushed and doesn't have a custom class?